boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -252,3 +265,10 @@ zephyr_udc0: &usbhs {
 	pinctrl-0 = <&can120_default>;
 	pinctrl-names = "default";
 };
+
+&pwm130 {
+	status = "okay";
```

Comment:
I think you should add `memory-regions = <&cpuapp_dma_region>;` here.

Reply:
I second that - without it `fade_led` sample does not work.

Reply:
Added